R/create_mofa_from_mae.R

```diff
@@ -11,7 +13,11 @@
 #' @param extract_metadata Boolean specifying whethet metadata should be extracted (default is FALSE).
 #' 
 #' @param ... additional parameters for MOFA, named as the options
-#'   of \code{\link[MOFA2:prepare_mofa]{prepare_mofa}}.
+#' of \code{\link[MOFA2:prepare_mofa]{prepare_mofa}}.
+#' \itemize{
+#' \item \code{altexps}: Specifies alternative experiments for each experiment.
```

Comment:
Maybe we could write a bit more explicitly that in case altExps are specified they will be used instead of assay.type? And if this option is mutually exclusive with assay.types argument? An example of this behaviour may be valuable.

Reply:
What do you think? Is the documentation now clear enough?